flocker/node/script.py

```diff
@@ -254,8 +258,30 @@ class ServeOptions(Options):
     """
     Command line options for ``flocker-serve`` cluster management process.
     """
-    # Maybe options for things like what port to listen on or perhaps where to
-    # find certificate material to use for TLS.
+    optParameters = [
+        ["port", "p", 4523, "The port to listen on.", int],
+        ]
+
+
+class _ServeService(MultiService):
+    """
+    Service for running a ``VolumeService`` and HTTP API service.
+    """
+    def __init__(self, volume_service, http_service):
+        """
+        :param volume_service: The volume service to run.
+        """
+        MultiService.__init__(self)
+        volume_service.setServiceParent(self)
+        http_service.setServiceParent(self)
+
+    def stopService(self):
+        """
+        :return: result from stopping the volume service.
```

Comment:
I didn't understand why you specifically return the volume service result? 
Maybe worth explaining in the docstring if it's important.